sklearn/cluster/_birch.py

```diff
@@ -579,10 +580,9 @@ def predict(self, X):
         """
         X = check_array(X, accept_sparse='csr')
         self._check_fit(X)
-        reduced_distance = safe_sparse_dot(X, self.subcluster_centers_.T)
-        reduced_distance *= -2
-        reduced_distance += self._subcluster_norms
```

Comment:
I meant that we could pass these sub cluster norms into pairwise_distances_argmin

Reply:
Something like this ?
```
kwargs = { ['Y_norm_squared']: self._subcluster_norms[np.newaxis, :]}
return self.subcluster_labels_[
                pairwise_distances_argmin(X, self.subcluster_centers_, metric_kwargs = kwargs)
            ]
```